src/App.js

```diff
@@ -375,28 +375,35 @@ class App extends Component {
       })
     }
 
+    // let provSetting = settings.get('should_provision')
+    // let should_prov = null;
+    //
+    // if (provSetting) {
+    //   let should_provision = provSetting.includes(boxID);
+    //
+    //   if (should_provision) {
+    //     should_prov = true;
+    //   } else {
+    //     should_prov = false;
+    //   }
+    // } else {
+    //   should_prov = false;
+    // }
+
```

Comment:
You can go ahead and delete this commented code :)

Reply:
Oops! My bad!